README.md

```diff
@@ -18,6 +18,30 @@ git clone https://github.com/redhat-et/copilot-ops.git
 # Build the binary
 make build
 ```
+To install the command line tool, complete the following steps:
+
+```console
+# Create bin directory
+mkdir -p ${HOME}/bin
+
+# Save binary to bin directory (run from copilot-ops directory)
+cp copilot-ops ${HOME}/bin
+
+# Make binary executable 
+chmod 755 ${HOME}/bin/copilot-ops
+
+# Open file ${HOME}/.bashrc in text editor, add line and save
```

Comment:
I prefer to avoid writing to bashrc and reloading it, and I think we can just give a general instruction here:
```suggestion
# Add the bin dir to path in your shell, or persist it by adding to the user's `~/.profile`:
```